tensorflow/core/kernels/sparse_utils.cc

```diff
@@ -225,10 +318,12 @@ Status ValidateSparseTensor(const Tensor& indices, const Tensor& values,
       const std::vector<TypeIndex>& row_start_indices);                     \
   template std::vector<TypeIndex> ParseRowStartIndices<TypeIndex>(          \
       const tensorflow::Tensor& tensor,                                     \
+      const TypeIndex num_nonzero_entries_in_sparse_mat);
```

Comment:
This line is the same as the next one, we don't need it (it causes compile error)